apps/wallet/src/background/accounts/zk/utils.ts

```diff
@@ -24,6 +24,41 @@ export function prepareZKLogin(currentEpoch: number) {
 	};
 }
 
+const forceSilentGetProviders: ZkProvider[] = ['twitch'];
+
+/**
+ * This method does a get request to the authorize url and is used as a workarround
+ * for `forceSilentGetProviders` that they do the silent login/token refresh using
+ * html directives or js code to redirect to the redirect_url (instead of response headers) and that forces the launchWebAuthFlow
+ * to open and close quickly a new window. Which closes the popup window when open but also creates a weird flickering effect.
+ *
+ * @param authUrl
+ */
+async function tryGetRedirectURLSilently(provider: ZkProvider, authUrl: string) {
```

Comment:
Without this the silent auth flow is not working as expected (opens and closes a window and forces the popup to close).

Is a bit hacky and hope it is ok from security perspective as well